packages/ui/nodes/link/src/PlateFloatingLink.tsx

```diff
@@ -45,6 +46,14 @@ export const PlateFloatingLink = () => {
           placeholder="Text to display"
         />
       </FloatingInputWrapper>
+      <div tw="h-px bg-gray-200" />
+
+      <FloatingInputWrapper>
+        <FloatingIconWrapper>
+          <FloatingLink.NewTabInput />
+        </FloatingIconWrapper>
+        <label css={floatingLabelCss}>Open link in new tab</label>
+      </FloatingInputWrapper>
```

Comment:
As this feature is not standard, we can remove it from this component.

This is a dumb component which is meant to be forked by the consumer if needed.